src/I8Beef.UniFi.Video/Protocol/Bootstrap/Bootstrap.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System.Collections.Generic;
+using I8Beef.UniFi.Video.Protocol.Common;
+
+namespace I8Beef.UniFi.Video.Protocol.Bootstrap
+{
+    /// <summary>
+    /// Bootstrap response.
+    /// </summary>
+    public class Bootstrap
+    {
+        /// <summary>
+        /// Admin user group id.
+        /// </summary>
+        public string AdminUserGroupId { get; set; }
+
+        // todo: alertSchedules
+
+        // todo: cameraSchedules
+
+        /// <summary>
+        /// Cameras
+        /// </summary>
+        public IEnumerable<Camera.Camera> Cameras { get; set; }
```

Comment:
IList to follow the rest of the collection standards?

Reply:
Agreed, will make this change.